clang/test/Misc/pragma-attribute-supported-attributes-list.test

```diff
@@ -144,6 +144,7 @@
 // CHECK-NEXT: SYCLDevice (SubjectMatchRule_function)
 // CHECK-NEXT: SYCLDeviceIndirectlyCallable (SubjectMatchRule_function)
 // CHECK-NEXT: SYCLIntelKernelArgsRestrict (SubjectMatchRule_function)
+// CHECK-NEXT: SYCLSimd (SubjectMatchRule_function)
```

Comment:
IMO no one would really use #pragma clang spelling in SYCL code, so I suggest to set PragmaAttributeSupport bit to zero in attribute definition in Attr.td

Reply:
Why do you think so? I suggest that this attribute to be consistent with the other  attributes above.

Reply:
It seems that perhaps ALL SYCL attributes should reject the pragma spelling then.  Can you file an issue to do so?

Reply:
I can file. Please provide justification why ALL SYCL attributes should reject the pragma spelling - I will add that to the issue.